src/Applications/NodeGossiper.py

```diff
@@ -44,12 +44,12 @@
 ###############################################################################
 NodeGossiper_module_aliases = {}
 for m in [
-    "os",
-    "subprocess",
+    "bcolors",
     "getopt",
     "math",
+    "os",
+    "subprocess",
     "sys",
```

Comment:
Good practice to use alphabetical order!

Reply:
Thanks :) Trying to implement as I edit files